plugins/state_history_plugin/state_history_plugin.cpp

```diff
@@ -20,6 +20,12 @@ namespace ws = boost::beast::websocket;
 
 extern const char* const state_history_plugin_abi;
 
+// during syncing if block is older than this, reduce logging 
+const int64_t   log_reduction_minutes = 5;    
```

Comment:
I actually think this makes it less readable. If you are going to have a constant then make it a `fc::microseconds`.
Same goes with `log_reduction_one_per`, I would prefer just using `1000` in the code.

Reply:
Agreed, doubt somebody ever will be changing these. Done.